### PR TITLE
cdktf: install zsh completion

### PR DESCRIPTION
**Pull request by stephengroat**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/cdktf.rb [hunk 1]
```diff
@@ -22,6 +22,9 @@ class Cdktf < Formula
   def install
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
     bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    output = Utils.safe_popen_read("#{bin}/cdktf", "completion", "--help")
```
Comment (SMillerDev) [inline Formula/cdktf.rb:26]: Doesn't this just output the help text?

**Review by SMillerDev — DISMISSED** [review]

File: Formula/cdktf.rb (hunk 1 above)
Comment (stephengroat) [reply Formula/cdktf.rb:26]: from testing, no, it seems like `--help` outputs the completion only

Comment (branchvincent) [reply Formula/cdktf.rb:26]: it accepts no arguments, however there's currently a bug causing strange behavior that this works around: https://github.com/hashicorp/terraform-cdk/issues/1905

it also supports both bash + zsh completions [by reading `SHELL`](https://github.com/yargs/yargs/blob/db354232705623bbcd8fad362f6a4d6d59650be5/lib/completion.ts#L44-L46), so we should add both. can be modeled after doitlive:

https://github.com/Homebrew/homebrew-core/blob/d034ddd0652e47f89a6e80a73d87c30d7f7913db/Formula/doitlive.rb#L57

Comment (stephengroat) [reply Formula/cdktf.rb]: addressed in 36e0da6e0c4aec900d94d1acd98161a7379abb2d

Comment (stephengroat) [reply Formula/cdktf.rb]: addressed in 4bf5a078330dd4abb156ec6b7c7209996761f340

**Review by branchv — APPROVED** [review]
@stephengroat thank you!

**Pull request closed without merging**